extensions/functions_comparison.yaml

```diff
@@ -107,4 +107,23 @@ scalar_functions:
           - value: fp64
         nullability: DECLARED_OUTPUT
         return: BOOLEAN
+  -
+    name: "if_else"
+    description: If true, return the first value, otherwise return the second.
```

Comment:
I'm a bit unsure about phrasing here; does it make sense to refer to the second and third parameters as "first" and "second" values? Perhaps not. Not sure how best to go about this.